projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid.interface.ts

```diff
@@ -57,3 +58,78 @@ export enum PivotDimensionType {
     Row,
     Column
 }
+
+// Equals to pretty much this configuration:
```

Comment:
Nice try skipping the proper format for the API docs ;)

Reply:
Do we have any way of writing descriptions for classes anyway?

Reply:
you can still leave a description on that class using the /** */ notation that will show up when the user tries typing `new IgxPivot...` etc . And add a sample usage if you like. Doesn't have to be super special.